### PR TITLE
created rg deployment

### PR DESCRIPTION
**Pull request by vermegi**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

Added simple resource group deployment template 
*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
I think we have an example of this covered in https://github.com/Azure/azure-quickstart-templates/tree/master/subscription-deployments/create-rg-lock-role-assignment which creates the group and rounds out the scenario a bit more.

**Comment by vermegi** [comment]
Well, indeed, there is a more complicated version of this one in the quickstart templates, however, I created this one for someone wanting to start out with just a base template. For the more elaborate one, you need to put in more effort, with creating the service principal, while this one just creates the resource group for you.

File: subscription-deployments/create-rg/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "parameters": {
+    "rgName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the resourceGroup to create"
+      }
+    },
+    "rgLocation": {
+      "type": "string",
+      "metadata": {
```
Comment (bmoore-msft) [inline subscription-deployments/create-rg/azuredeploy.json:13]: add a defaultValue of: "[deployment().location]"

File: subscription-deployments/create-rg/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "description": "Location for the resourceGroup"
+      }
+    }
+  },
+  "variables": { },
```
Comment (bmoore-msft) [inline subscription-deployments/create-rg/azuredeploy.json:18]: remove empty variables property

File: subscription-deployments/create-rg/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "rgName": {
+      "value": "GEN-UNIQUE"
+    },
+    "rgLocation": {
```
Comment (bmoore-msft) [inline subscription-deployments/create-rg/azuredeploy.parameters.json:8]: remove this when you add the defaultValue

File: subscription-deployments/create-rg/metadata.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "Create a resourceGroup",
+    "description": "This template is a subscription level template that will create a resourceGroup.  Currently, this template cannot be deployed via the Azure Portal.",
```
Comment (bmoore-msft) [inline subscription-deployments/create-rg/metadata.json:4]: update the description - this can be deployed via the portal

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: subscription-deployments/create-rg/azuredeploy.json (hunk 1 above)
Comment (vermegi) [reply subscription-deployments/create-rg/azuredeploy.json:13]: done

File: subscription-deployments/create-rg/azuredeploy.json (hunk 2 above)
Comment (vermegi) [reply subscription-deployments/create-rg/azuredeploy.json]: done

File: subscription-deployments/create-rg/azuredeploy.parameters.json (hunk 3 above)
Comment (vermegi) [reply subscription-deployments/create-rg/azuredeploy.parameters.json]: removed rgLocation. rgName is still there.

**Pull request merged**